ui.tests/test-module/specs/textinput/textinput.authoring.spec.js

```diff
@@ -170,9 +169,8 @@ describe('Page - Authoring', function () {
     //}
 
     // not yet in available publicly released april far
-    //if (cy.af.isLatestAddon()) {
-      // todo: needs to be uncommented later
-      it.skip('Test z-index of Rule editor iframe for components inside site container', function () {
+    if (cy.af.isLatestAddon()) {
```

Comment:
Can you remove isLatestAddon check, since these changes are already released ?

Reply:
Done.